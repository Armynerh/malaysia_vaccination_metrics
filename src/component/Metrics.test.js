import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Metrics from './Metrics';

test('Metrics component snapshot', () => {
  const { asFragment } = render(
    <Provider store={store}>
      <Metrics />
    </Provider>,
  );

  expect(asFragment()).toMatchSnapshot();
});
