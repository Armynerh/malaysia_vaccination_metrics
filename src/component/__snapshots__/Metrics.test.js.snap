// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Metrics component snapshot 1`] = `
<DocumentFragment>
  <div>
    Loading...
  </div>
</DocumentFragment>
`;
